### PR TITLE
2694 selenium c status bar not removed on iphone 12 safari

### PR DESCRIPTION
**Pull request by at-ylazarenko**
Support for iOS 15

### CODE REVIEW

File: Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs [hunk 1]
```diff
@@ -9,7 +9,8 @@ namespace Applitools.Selenium.Capture
 {
     internal class SafariScreenshotImageProvider : MobileScreenshotImageProvider
     {
-        private static Dictionary<Size, Rectangle[]> devicesRegions_ = null;
+        private static IDictionary<string, IDictionary<Size, Rectangle[]>> regionsPeriOsVersion_;
+        private static string defaultiOSKey = "*";
```
Comment (itaibh) [inline Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs]: could be made readonly

File: Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs [hunk 2]
```diff
@@ -173,9 +193,18 @@ private static void InitDeviceRegionsTable_()
 
                 { new Size(1640, 2360), new Rectangle[]{ new Rectangle(0, 149, 1640, 2211) } },
 
-                // iPhone 12
+                // iPhone 12, iOS 14
                 { new Size(1170, 2532), new[]{ new Rectangle(0, 291, 1170, 1991)} },
                 { new Size(2532, 1170), new[]{ new Rectangle(141, 150, 2250, 981)} },
+            }},
```
Comment (itaibh) [inline Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs]: fix indentation

**Review by itaibh — CHANGES_REQUESTED** [review]

File: Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs (hunk 2 above)
Comment (at-ylazarenko) [reply Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs]: done

File: Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs (hunk 1 above)
Comment (at-ylazarenko) [reply Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs]: made a const

**Review by itaibh — APPROVED** [review]

**Pull request merged**